sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_models.hpp

```diff
@@ -802,4 +802,34 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
     std::string Version;
   };
 
+  /**
+   * @brief Parameters for StartCreateCErtificate
+   *
+   */
+  struct CertificateCreateParameters final
+  {
+    /**
+     * @brief Certificate policy
+     *
+     */
+    CertificatePolicy Policy;
+
+    /**
+     * @brief certificate attributes
+     *
+     */
+    CertificateProperties Properties;
+
+    /**
+     * @brief enabled?
```

Comment:
This as well, not sufficient. Maybe it is enough to comply with the policy, but we can't call this "documented" :)